app/views/events/_event.html.erb

```diff
@@ -2,7 +2,9 @@
   <td><%= event.name %></td>
   <td>
     <%= link_to 'Show', event_path(event) %>
-    <%= link_to 'Edit', edit_event_path(event) %>
-    <%= render partial: 'delete-link', locals: { event: event } %>
+    <% if current_person&.admin? %>
```

Comment:
Yes that makes sense - currently only an admin is permitted to edit/delete events, but if that changes in future, we wouldn't have to update the view.